tests/pipelines/text_to_video/test_text_to_video_zero_sdxl.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# Copyright 2023 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import torch
+
+from diffusers import DDIMScheduler, TextToVideoZeroSDXLPipeline
+from diffusers.utils import require_torch_gpu, slow, torch_device
+
+
+class TextToVideoZeroSDXLPipelineFastTests(unittest.TestCase):
+    def test_forward_loop(self):
```

Comment:
Hey there @patrickvonplaten, when subclassing from `PipelineTesterMixin`, most of the tests fail because `pipeline.unet.set_default_attn_processor` cannot be called as our custom attention processor is not in `CROSS_ATTENTION_PROCESSORS` or `ADDED_KV_ATTENTION_PROCESSORS`. What is the correct workaround here? Should I add our custom attention processor in `diffusers.models.attention_processor` or should I inherit the tests from the parent and manually set the attention processor?

Reply:
or should I get rid of `PipelineTesterMixin` instead?

Reply:
Ah no, we can just overwrite the test here as done here for example: https://github.com/huggingface/diffusers/blob/89d8f84893ce7ba957a51bc1f48c608c564a5e69/tests/pipelines/text_to_video/test_video_to_video.py#L189